main.go

```diff
@@ -164,7 +167,38 @@ func (s *admissionWebhookServer) createVolumesPatch(p string, volumes []corev1.V
 	return jsonpatch.NewOperation("add", path.Join(p, "volumes"), volumes)
 }
 
+func parseSRIOVLabels(v string, logger *zap.SugaredLogger) map[string]int {
+	networkServices := []*url.URL{}
+	poolResources := make(map[string]int)
+
+	for _, rawNetNS := range strings.Split(v, ",") {
+		netNS, err := url.Parse(rawNetNS)
+
+		if err != nil {
+			logger.Errorf("Malformed NS annotation: %+v", rawNetNS)
+			return nil
+		}
+		networkServices = append(networkServices, netNS)
+	}
+
+	for _, networkService := range networkServices {
+		u := (*nsurl.NSURL)(networkService)
+		labels := u.Labels()
+		if _, ok := labels["sriovToken"]; ok {
+			interfacePools := strings.Split(labels["sriovToken"], ",")
```

Comment:
@Bolodya1997 Do we have a const for sriovToken?

Reply:
@denis-tingaikin we have this:

https://github.com/networkservicemesh/sdk-sriov/blob/main/pkg/networkservice/common/token/client.go#L41

but the constant starts with a lower case letter so it is not exported and golang doesn't allow to use it.

Reply:
It sounds to me like the const should be exported to avoid hardcoded strings in other places.
@Bolodya1997 Thoughts?

Reply:
It is actually mostly a contract than dependency, because the most common usage of this constant is manually using it in `.yaml` file - in such case there is no way to import it.
But it can be changed to exported to be used here as a dependency, probably it would be better to do such thing.

Reply:
@denis-tingaikin  @Bolodya1997 If there is no other objection maybe we can proceed with the merging of the PR as it is and when the variable gets exported correctly then we can change the "sriovToken" string in a later PR. What do you think?

Reply:
@mardim91 I think it should be easy to export the constant from the package. Could you open the PR for sdk-sriov? We will consider it as soon as possible.

Reply:
@denis-tingaikin I can do that yes. The only thing that I will do is upper case the vars

Reply:
@mardim91 Finally I think it can be considered by separate PRs.

Reply:
@denis-tingaikin What do you mean it can be considered by separate PRs ? I didn't get that